File: Day-14/Pattern20.py
n = int(input())

def PatternTwenty(n):
    i = 1
    while(i <=n):
        star = n-i+1
        while(star):
            print('*', end='')
            star -= 1
        i += 1
        print(end='\n')
    
PatternTwenty(n)

File: Day-15/Pattern20.js
function PatternTwenty(n) {
  let i = 0;
  while (i < n) {
    let star = n - i;
    let str = "";
    while (star) {
      str += "*";
      star--;
    }
    console.log(str);
    i++;
  }
}

PatternTwenty(4);
